sklearn/ensemble/voting_classifier.py

```diff
@@ -252,17 +269,29 @@ def transform(self, X):
         else:
             return self._predict(X)
 
+    def set_params(self, **params):
+        """ Setting the parameters for the voting classifier
+
+        Valid parameter keys can be listed with get_params().
+
+        Parameters
+        ----------
+        params: keyword arguments
+            Specific parameters using e.g. set_params(parameter_name=new_value)
+            Estimators can be removed by setting them to None. In the following
+            example, the RandomForestClassifier is removed:
+            clf1 = LogisticRegression()
```

Comment:
I am fairly sure that you have to add this under a side-heading "Examples" so that it renders properly